pkg/oci/utils.go

```diff
@@ -23,24 +23,37 @@ import (
 	"strconv"
 	"strings"
 
+	criContainerdAnnotations "github.com/containerd/cri-containerd/pkg/annotations"
 	vc "github.com/containers/virtcontainers"
 	vcAnnotations "github.com/containers/virtcontainers/pkg/annotations"
-	"github.com/kubernetes-incubator/cri-o/pkg/annotations"
+	crioAnnotations "github.com/kubernetes-incubator/cri-o/pkg/annotations"
 	spec "github.com/opencontainers/runtime-spec/specs-go"
 	"github.com/sirupsen/logrus"
 )
 
+type annotationContainerType struct {
+	annotation    string
+	containerType vc.ContainerType
+}
+
 var (
 	// ErrNoLinux is an error for missing Linux sections in the OCI configuration file.
 	ErrNoLinux = errors.New("missing Linux section")
 
 	// CRIContainerTypeKeyList lists all the CRI keys that could define
 	// the container type from annotations in the config.json.
-	CRIContainerTypeKeyList = []string{annotations.ContainerType}
+	CRIContainerTypeKeyList = []string{criContainerdAnnotations.ContainerType, crioAnnotations.ContainerType}
 
 	// CRISandboxNameKeyList lists all the CRI keys that could define
 	// the sandbox ID (pod ID) from annotations in the config.json.
-	CRISandboxNameKeyList = []string{annotations.SandboxID}
+	CRISandboxNameKeyList = []string{criContainerdAnnotations.SandboxID, crioAnnotations.SandboxID}
+
+	CRIContainerTypeList = []annotationContainerType{
```

Comment:
You need to add a comment for that one, as it's an exported variable.

Reply:
Comment added.